assets/app/scripts/filters/util.js

```diff
@@ -156,12 +156,27 @@ angular.module('openshiftConsole')
     };
   })
   .filter('githubLink', function() {
-    return function(link, commit) {
+    return function(link, ref, contextDir) {
       var m = link.match(/^(?:https?:\/\/|git:\/\/|git\+ssh:\/\/|git\+https:\/\/)?(?:[^@]+@)?github\.com[:\/]([^\/]+\/[^\/]+?)(?:\.git(#.*)?)?$/);
       if (m) {
         link = "https://github.com/" + m[1];
-        if (commit) {
-          link += "/commit/" + commit;
```

Comment:
The existing regex is wrong, it doesnt match if the URL has a trailing slash, i'll fix that and make sure that m[1] doesn't include the /